includes/Admin/Orders.php

```diff
@@ -88,9 +88,11 @@ public function enqueue_assets() {
 		], \WC_Facebookcommerce::VERSION );
 
 		wp_localize_script( 'wc-facebook-commerce-orders', 'wc_facebook_commerce_orders', [
-			'order_id'          => $order->get_id(),
-			'is_commerce_order' => Commerce\Orders::is_commerce_order( $order ),
-			'shipment_tracking' => $order->get_meta( '_wc_shipment_tracking_items', true ),
+			'order_id'                  => $order->get_id(),
+			'order_status'              => $order->get_status(),
```

Comment:
Out of curiosity, why you decided not to use the value of the select on document ready, instead of adding this param?

Reply:
Well, to execute any related code as soon as the page load without depending on the select input as that might change on page load for any reason by another code from different plugin

Reply:
I do not believe it to be necessary...

Reply:
I don't have strong feeling about it ... I just thought it would be more efficient and easier to change in the future